test/e2e/servers/ory/kratos.spec.ts

```diff
@@ -633,3 +633,40 @@ describe("updates user phone", () => {
     await disposeAdminClient()
   })
 })
+
+describe("cookie flow", () => {
+  const authService = AuthWithPhonePasswordlessService()
+  const phone = randomPhone()
+  let cookies: Array<KratosCookie>
+  let cookieStr = ""
+
+  it("login with cookie", async () => {
+    const res = await authService.createIdentityWithCookie(phone)
+    if (res instanceof Error) throw res
+    expect(res).toHaveProperty("kratosUserId")
+    expect(res).toHaveProperty("cookiesToSendBackToClient")
+    cookies = res.cookiesToSendBackToClient
+  })
+
+  it("logout and revoke cookies", async () => {
+    for (const cookie of cookies) {
```

Comment:
this create a conditional test.. it should be independent of `login with cookie`